lib/DDGC/DB/Result/User/Blog.pm

```diff
@@ -1,202 +1,108 @@
 package DDGC::DB::Result::User::Blog;
+
 # ABSTRACT: Result class of blog posts
 
-use Moose;
-use MooseX::NonMoose;
-extends 'DDGC::DB::Base::Result';
+use Moo;
```

Comment:
This is  basically a temporary table def because notifications still rely on it - I was too hasty moving it, and reinstated a minimal version just for column definitions.